.github/workflows/sdk.yml

```diff
@@ -24,6 +24,13 @@ jobs:
           path: 'sdk'
       - name: Copy API definition
         run: cp service/src/openapi.json sdk/openapi.json
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
```

Comment:
I am used to Azul Zulu because that used to be the best (only) OpenJDK tested against the TCK. But I see Azul actually supports Temurin and Temurin ticks all the boxes. Nice.